pyproject.toml

```
@@ -1,40 +1,55 @@
-[tool.poetry]
+[project]
+authors = [
+    {name = "Roman Right", email = "roman-right@protonmail.com"},
+]
+license = {text = "Apache-2.0"}
+requires-python = "<4.0,>=3.9"
```

I would revert this since it reads more naturally to me.
```suggestion
requires-python = ">=3.9,<4.0"
```